src/styles/fontFamily/index.js

```diff
@@ -7,9 +7,9 @@ const fontFamily = {
     EXP_NEUE: 'ExpensifyNeue-Regular',
```

Comment:
Sorry for thinking of this so late... but I guess semantically, it makes more sense to just call this "Expensify Neue" or in this case, `'ExpensifyNeue'` as that's the name of the font family. "Regular" is simply a weight of the font, like Bold or Semi-Bold or Light, etc. No need to fix this immediately, but just wanted to jot my thoughts down.

Reply:
I agree! could we bundle this with the new kansas changes?

Reply:
I'm down with that@ cc @luacmartins as I saw you picked up the NK changes.

Reply:
Just FYI we created a separate issue for this change: https://github.com/Expensify/App/issues/13772